lib/xcpretty/formatters/simple.rb

```diff
@@ -27,6 +27,10 @@ def format_build_target(target, project, configuration)
       format("Building", "#{project}/#{target} [#{configuration}]")
     end
 
+    def format_analyze_target(target, project, configuration)
+      format("Building and Analyzing", "#{project}/#{target} [#{configuration}]")
```

Comment:
Line is too long. [81/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.